pkg/rpc/response/result/asset_state.go

```diff
@@ -16,8 +16,8 @@ type AssetState struct {
 	Amount     util.Fixed8           `json:"amount"`
 	Available  util.Fixed8           `json:"available"`
 	Precision  uint8                 `json:"precision"`
-	FeeMode    uint8                 `json:"fee"`
-	FeeAddress util.Uint160          `json:"address"`
+	FeeMode    uint8                 `json:"-"`
+	FeeAddress util.Uint160          `json:"-"`
```

Comment:
Why not just drop them?

Reply:
And make a separate commit, because it's a separate change.

Reply:
Done